### PR TITLE
My plugin

### PR DESCRIPTION
**Pull request by himanshusharma1**

### CODE REVIEW

**Comment by himanshusharma1** [comment]
How can I merge it??

File: files/kalturahtml5plugin/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Himanshu Sharma"
+github = "https://github.com/himanshusharma1"
```
Comment (jimaek) [inline files/kalturahtml5plugin/info.ini]: This needs to be a link to the project itself

**Comment by pnommensen** [comment]
The use case here seems very limited and I do not see the source files. Can you link them and explain where it's being used? Thanks.

**Comment by pnommensen** [comment]
ping @himanshusharma1

**Pull request closed without merging**